org.eclipse.scout.rt.mail.test/src/test/java/org/eclipse/scout/rt/mail/MailIDNConverterTest.java

```diff
@@ -69,4 +69,18 @@ public void testUnicodeConvertionWithSpecialChar() {
     // Assert
     assertEquals(emailAddress, convertedAddress);
   }
+
+  @Test
+  public void testLongDomainNames() {
+    // sample e-mail address with long domain name
+    String validLongEmail = "nobody@abcdefghijklmnopqrstuvwxyabcdefghijklmnopqrstuvwxyabcdefghijklm.de";
+    assertEquals(validLongEmail, BEANS.get(MailIDNConverter.class).toASCII(validLongEmail));
```

Comment:
local variable MailIDNConverter converter = BEANS.get(MailIDNConverter.class); and use 3x

Reply:
Thanks, fixed in 339e2717ef0a655f0c48eeb8d88f93efef8cf20a